### PR TITLE
Translated the existing doc done from french to english + minor french corrections

### PR DESCRIPTION
**Pull request by CypherJack**
You might verify that I did stuff correctly enough as I am not really familiar with GitHub.
I think it's okay maybe the commits could have been done in one but I did on the go not knowing how to use git I then used the web app.

### CODE REVIEW

File: docs/fr/delivery-management.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Mon panier contient 4 lignes articles.
 Dans le cadre d'une configuration "Flat fees" de 10€, le prix des frais de port sera de 10€.
 Dans le cadre d'une configuration "Flat fees" de 10€ avec l'option "Apply flat fee for each item instead of once for the whole order", le prix des frais de port sera de 4 lignes articles * 10€, soit 40€.
 
-### Article Standalone
+### Article Indépendant
```
Comment (Tirodem) [inline docs/fr/delivery-management.md:33]: "Standalone" (even in french) is a specific e-commerce wording, we keep it as it is.
You can remove this correction.

File: docs/fr/point-of-sale-options.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Point of Sale option
+# Option de point de vente (PoS)
```
Comment (Tirodem) [inline docs/fr/point-of-sale-options.md:1]: Same as Standalone
You can add ("Point of sale = Point de vente") if you want to explain, but PdV isn't use in french e-commerce, where PoS is a lot.

File: docs/fr/point-of-sale-options.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ En cas d'erreur d'ajout panier, l'erreur sera notifiée et le champ Alias sera v
 
 ![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/376b83c9-29fd-485a-8b5d-dccfa1f97813)
 
-A noter qu'en cas d'ajout d'article PWYW via alias, le montant du produit sera le montant minimum configuré sur le produit.
+A noter qu'en cas d'ajout d'article prix libre via alias, le montant du produit sera le montant minimum configuré sur le produit.
```
Comment (Tirodem) [inline docs/fr/point-of-sale-options.md:87]: Je préfère un "en cas d'ajout d'article à prix libre (PWYW)"

**Review by Tirodem — CHANGES_REQUESTED** [review]
Check comments

**Comment by CypherJack** [comment]
Updated

**Review by Tirodem — APPROVED** [review]

**Pull request merged**